lldb/source/Target/StackFrame.cpp

```diff
@@ -508,6 +508,32 @@ StackFrame::GetInScopeVariableList(bool get_file_globals,
 ValueObjectSP StackFrame::GetValueForVariableExpressionPath(
     llvm::StringRef var_expr, DynamicValueType use_dynamic, uint32_t options,
     VariableSP &var_sp, Status &error) {
+  // Check to see if we should use the DIL implementation.
+  lldb::TargetSP target_sp = CalculateTarget();
+  ExecutionContext exe_ctx;
+  CalculateExecutionContext(exe_ctx);
+  bool use_DIL = target_sp->GetUseDIL(&exe_ctx);
+  if (use_DIL)
+    return DILGetValueForVariableExpressionPath(var_expr, use_dynamic, options,
+                                                var_sp, error);
+  else
```

Comment:
https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

(I also think the comments in this method don't tell us anything that's not immediately obvious from the code. I'd just remove them.)

Reply:
Done.